binaries/speed_benchmark_torch.cc

```diff
@@ -198,6 +200,7 @@ int main(int argc, char** argv) {
     std::cout << module.forward(inputs) << std::endl;
   }
 
+  c10::WithCPUCachingAllocatorGuard cachine_allocator_guard;
```

Comment:
We should probably make a flag to disable this.

Reply:
You mean ifdef for mobile only visibility?